src/systems/userinput/bindings/oculus-go-user.js

```diff
@@ -76,10 +76,10 @@ export default function generate3DOFTriggerBindings(device) {
           west: dpadWest,
           center: dpadCenter
         },
-        xform: xforms.vec2dpad(0.8)
+        xform: xforms.vec2dpad(0.3)
       },
       {
-        src: [dpadNorth, dpadSouth, dpadCenter],
```

Comment:
Hmm, this also affects freeze/thaw... does the binding using `dpadNorth` (below) mask this minding? If so, I think you don't need to remove `dpadNorth` and `dpadCenter` and freeze/thaw will continue to work as it does now when not holding the pen.